documentation/Database-Schema-Template.md

```diff
@@ -33,37 +33,42 @@ The `version_id` is the link with the template version.
 
 The `template_name` and `code` is to help admin users identifying one template.
 
-The `current_status` can be `'Draft'`, `'Available'` or `'Disabled'`. The only 'Available' templates are the onesthe in the version flagged as the current one. All other application template are 'Disabled' or 'Draft' (if unfinalised).
+**To be re-considered after issue #35 is in PR or merged to master:**
+~~The `current_status` can be `'Draft'`, `'Available'` or `'Disabled'`. The only 'Available' templates are the ones in the version flagged as the current one. All other application template are 'Disabled' or 'Draft' (if unfinalised).~~
 
 ### template stage
 
 There is one or more stages per application template. Each one is defined as a new record that point to `template_id` with the `number` of this stage (1, 2, 3, ...) and title can be anything, but typically would be: `'Screening'`, `'Assessment'`, `'Final Stage'`.
 
 ### template sections
 
-Sections of the application template that contain some elements. Each section can be associated with a **template permission** if this section requires a certain type of reviewer to check for responses from an Applicant.
+Sections of the application template that contain some [elements/questions](Elements-Questions.md). Each section can be associated with a **template permission** if this section requires a certain type of reviewer to check for responses from an Applicant.
 
 ### template elements
 
 Elements in the application template always are part of a section, therefor it stores the `section_id` instead of the `template_id`.
```

Comment:
```suggestion
Elements in the application template always are part of a section, therefore it stores the `section_id` instead of the `template_id`.
```